tutorials/plot_background_filtering.py

```diff
@@ -929,7 +929,7 @@ def baseline_plot(x):
 # ``h_freq + transition_bandwidth/2.``.
 #
 # Filter length (order) and transition bandwidth (roll-off)
-# --------------------------------------------------------
+# ---------------------------------------------------------
```

Comment:
Try to avoid cosmetic changes. See point 4 [here](http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/).